1.20/Dockerfile.fedora

```diff
@@ -59,19 +59,17 @@ COPY ./root/ /
 # writable as OpenShift default security model is to run the container under
 # random UID.
 RUN sed -i -f ${NGINX_APP_ROOT}/nginxconf-fed.sed ${NGINX_CONF_PATH} && \
-    chmod a+rwx ${NGINX_CONF_PATH} && \
     mkdir -p ${NGINX_APP_ROOT}/etc/nginx.d/ && \
     mkdir -p ${NGINX_APP_ROOT}/etc/nginx.default.d/ && \
     mkdir -p ${NGINX_APP_ROOT}/src/nginx-start/ && \
     mkdir -p ${NGINX_CONTAINER_SCRIPTS_PATH}/nginx-start && \
     mkdir -p ${NGINX_LOG_PATH} && \
     mkdir -p ${NGINX_PERL_MODULE_PATH} && \
+    chmod a+rwx ${NGINX_CONF_PATH} && \
     chmod -R a+rwx ${NGINX_APP_ROOT}/etc && \
-    chmod -R a+rwx /var /run && \
+    chmod -R a+rwx ${NGINX_APP_ROOT}/src/nginx-start/  && \
     chmod -R a+rwx ${NGINX_CONTAINER_SCRIPTS_PATH}/nginx-start && \
-    chown -R 1001:0 ${NGINX_APP_ROOT} && \
```

Comment:
This is needed because of at the end we are switching to USER 1000
https://github.com/sclorg/nginx-container/blob/master/1.20/Dockerfile.fedora#L78

Reply:
It is not.

1) Our tests run with user 10001, not 1001 and they work. However, I do not know, if that was intended: https://github.com/sclorg/nginx-container/blob/59508a8dc90458916926e5ff63762e8ab651cbc4/test/run#L72

2) the `${NGINX_APP_ROOT}` only has src/nginx-start and etc/ directories, and they have been already chmoded to 777.

3) As written in commit message, chown is IMO useless for use in openshift.

Reply:
Yeah, it was intended. By default, OpenShift runs a container as an arbitrary user. Like in our case. The number is or can be 100004557 or something like that.

Reply:
Now I see that we also chmod 777 the directories inside. Maybe we should revisit if it is really necessary to give permissions to everyone?

Reply:
I suppose, this is because we do not know, under which user the container is to be run in openshift environment.

Reply:
We know that the user is part of the root group, hence why the `chown -R 1001:0`. But I think the chmod 777 commands were added historically before we had that guarantee? Maybe @hhorak would remember

Reply:
Hmm, isn't GID also assigned randomly? [1]

[1] https://cloud.redhat.com/blog/a-guide-to-openshift-and-uids

Reply:
From the document you linked: "The Container user is always a member of the root group, so it can read or write files accessible by GID=0."
So I do not think anything changed there.

Reply:
Oh, I see, thanks for noticing. 
That changes the situation. I need to think about the proceeding then.

Reply:
I have proposed new solution.